src/timers/CTimers.cpp

```diff
@@ -0,0 +1,152 @@
+/// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (C) 2023 The MMapper Authors
+
+#include "CTimers.h"
+
+#include <QMutexLocker>
+#include <QSettings>
+#include <QTimer>
+
+CTimers::CTimers(QObject *parent)
+    : QObject(parent)
+{
+    m_nextId = 1;
+}
+
+CTimers::~CTimers() {}
+
+void CTimers::addTimer(QByteArray name, QByteArray desc)
```

Comment:
We can avoid making copies of the arguments here.

Reply:
:+1:

Reply:
I switched to std::strings and using const references. 
Saw one issue with latin-1 encoding, but that got to be my conversions somewhere...